community/mtd-utils/APKBUILD

```diff
@@ -0,0 +1,40 @@
+# Maintainer: Olliver Schinagl <oliver@schinagl.nl>
+pkgname=mtd-utils
+pkgver=2.1.0
+pkgrel=0
+pkgdesc="Utilities for handling MTD devices, and for dealing with FTL, NFTL JFFS2, etc."
+url="http://www.linux-mtd.infradead.org/"
+arch="all"
+license="GPL-2.0"
+makedepends="
+	acl-dev
+	linux-headers
+	lzo-dev
+	openssl-dev
+	util-linux-dev
+	zlib-dev
+"
```

Comment:
```suggestion
    "
```

Reply:
I don't get that; would you write:
```sh
if () {
    do_stuff
      }
```

Reply:
Just as you don't put the other continued lines in the first column I don't expect to see the ending quote there.